tests/utils/test_geometry_comparisons.py

```diff
@@ -217,6 +219,24 @@ def test_compute_relative_rotation_angle(self):
 
         np.testing.assert_allclose(computed_deg, expected_deg, rtol=1e-3, atol=1e-3)
 
+    def test_compute_relative_rotation_angle_euler(self) -> None:
+        """"Sample 100 rotations about each individual Euler angle, and ensure Scipy and GTSAM result match."""
+        num_samples = 100
```

Comment:
I think we can reduce num_samples. I found the scipy rotation function to be very time consuming when I used it before. I think num_samples = 10 or 5 should suffice.

Reply:
Interesting -- maybe they've pushed some optimizations since.

The test completes in 0.0569 sec (56 milliseconds), so I think the speed is ok currently.